### ISSUE TITLE
TensorBoard dones't work well on win10: 

### ISSUE THREAD

**Issue by dcscodelife** [tensorflow/tensorflow#7087]
events.out.tfevents.1485424988 has been generated，its default been encoded with ANSI (check with notepad++)
but tensroboard decode it with some problems as bellow:

> **E:\>tensorboard --inspect --logdir=e:/**
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cublas64_80.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cudnn64_5.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cufft64_80.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library nvcuda.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library curand64_80.dll locally
> 
> Processing event files... (this can take a few minutes)
> 
> 
> Traceback (most recent call last):
>   File "d:\users\dc\anaconda3\lib\runpy.py", line 184, in _run_module_as_main
>     "__main__", mod_spec)
>   File "d:\users\dc\anaconda3\lib\runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "d:\Users\dc\Anaconda3\Scripts\tensorboard.exe\__main__.py", line 9, in <module>
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\tensorboard\tensorboard.py", line 99, in main
>     efi.inspect(logdir, event_file, FLAGS.tag)
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\summary\event_file_inspector.py", line 409, in inspect
>     inspection_units = get_inspection_units(logdir, event_file, tag)
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\summary\event_file_inspector.py", line 360, in get_inspection_units
>     for subdir in subdirs:
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\summary\event_multiplexer.py", line 404, in <genexpr>
>     subdir
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\summary\impl\io_wrapper.py", line 50, in ListRecursively
>     for dir_path, _, filenames in gfile.Walk(top):
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\lib\io\file_io.py", line 446, in walk
>     for subitem in walk(os.path.join(top, subdir), in_order):
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\lib\io\file_io.py", line 427, in walk
>     listing = list_directory(top)
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\lib\io\file_io.py", line 406, in list_directory
>     compat.as_bytes(dirname), status)
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\lib\io\file_io.py", line 405, in <listcomp>
>     for filename in pywrap_tensorflow.GetChildren(
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\util\compat.py", line 106, in as_str_any
>     return as_str(value)
>   File "d:\users\dc\anaconda3\lib\site-packages\tensorflow\python\util\compat.py", line 84, in as_text
>     return bytes_or_text.decode(encoding)
> **UnicodeDecodeError: 'utf-8' codec can't decode byte 0xbb in position 2: invalid start byte**
> 
> E:\>

**Comment by KeyYD** [comment]
Did you use Chinese in your file but without a comment, #coding=utf-8,  at the start of it? If not so, I don't think that Window system is suitable for tensor board. You could try Ubuntu, Linux or Mac System

**Comment by KeyYD** [comment]
In Linux or Mac, you will get a link for tensorboard to watch it. But I am not sure whether it works in window

**Comment by drpngx** [comment]
@dandelionmane maybe issues with BOM on windows?

**Comment by drpngx** [comment]
@dcscodelife could you attach the events file?

**Comment by Franck-Dernoncourt** [comment]
@KeyYD on Microsoft Windows you also get a link when TensorBoard starts.

**Comment by dandelionmane** [comment]
@mrry for thoughts on windows issues?

**Comment by mrry** [comment]
Is the problem maybe that the logdir is `e:/` (which TensorBoard will interpret as a run called `'e'` and the path `\` on the same drive as the current working directory)? It looks like it's walking the entire filesystem, which is probably not what you wanted to happen.

Then I guess it's running into a filename that isn't UTF-8, perhaps because the filename handling code isn't Unicode aware?

As a workaround, I'd suggest putting your logs somewhere other than the root directory of your `E:` drive, so that TensorBoard doesn't encounter a file with a name it can't handle.

**Comment by dcscodelife** [comment]
@drpngx Thank you everybody, sorry I have not followed this issue until today. 
That day later it does work for me, I remember that I made a new folder 'log' in E:\, then it worked.
Firstly, I give up do it under windows and try it on ubuntu with the file generated under windows, then it worked, so I think the file is ok, so I think maybe some others things I should try, then I tried something according to tutorial under windows, likely checked some format of command or folder path or name, then it worked.
But I forget the details, but tensorflow becomes more strongly.
